Lib/test/test_asyncio/test_events.py

```diff
@@ -2701,37 +2701,6 @@ def test_event_loop_policy(self):
         self.assertRaises(NotImplementedError, policy.set_event_loop, object())
         self.assertRaises(NotImplementedError, policy.new_event_loop)
 
-    def test_get_event_loop(self):
-        policy = asyncio.DefaultEventLoopPolicy()
-        self.assertIsNone(policy._local._loop)
-        with self.assertWarns(DeprecationWarning) as cm:
-            loop = policy.get_event_loop()
-        self.assertEqual(cm.filename, __file__)
-        self.assertIsInstance(loop, asyncio.AbstractEventLoop)
-
-        self.assertIs(policy._local._loop, loop)
-        self.assertIs(loop, policy.get_event_loop())
-        loop.close()
-
-    def test_get_event_loop_calls_set_event_loop(self):
```

Comment:
You can also leave this test, but rename it to test_get_event_loop_does_not_call_set_event_loop.

Reply:
fixed